transforms/dialog-full-screen-app-wrapper/__testfixtures__/Basic.output.js

```diff
@@ -0,0 +1,5 @@
+import DialogFullScreen from "carbon-react/lib/components/dialog-full-screen";
+
```

Comment:
Does the codemod add in a line break before the import? If it does I reckon it shouldn't

Reply:
Yeah it does atm, I'll see if I can remove it, assumed prettier/ eslint would fix that when the consuming project runs them but it might miss that

Reply:
I had a further look into it and I believe this is the intended `recast` functionality where it reads the `\n` after the import and the next `\n` for the empty line in the input file. It doesn't add the empty lines unless this scenario is met so I'm not sure it's such a big deal.